source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsController.cs

```diff
@@ -86,17 +85,17 @@ public ActionResult PeriodEvaluation(string period)
                 return new HttpStatusCodeResult(403, "Access Denied");
             }
 
-            var isEvaluated = evaluation.Any(p => p.UserName == loggedUser);
-            var isEvaluator = evaluation.Any(p => p.ResponsibleId == loggedUser || (p.Evaluators != null && p.Evaluators.Contains(loggedUser)));
+            var userPeriods = RavenSession
+                .Query<Period_Serch.Projection, Period_Serch>()
+                .Statistics(out stats)
+                .Where(e => (e.UserName == loggedUser))
+                .OrderByDescending(e => e.Period)
+                .ToList();
```

Comment:
Some [lines ago](https://github.com/CommonJobs/CommonJobs/pull/476/files#diff-48f065f3d88335da2dce595120021376R83) it is checkin if the _period_ is available for this user. Do you think that we could use only one query to database in place of two?